pennylane/transforms/transpile.py

```diff
@@ -72,8 +70,8 @@ def transpile(
 
     Args:
         tape (QNode or QuantumTape or Callable): A quantum tape.
-        coupling_map (list[tuple(int, int)] or nx.Graph): Either a list of tuples(int, int) or an instance of
-            `networkx.Graph` specifying the couplings between different qubits.
+        coupling_map: Data specifying the couplings between different qubits. This data can be any format accepted by ``nx.to_networkx_graph()``,
```

Comment:
Maybe add a link to the documentation of `nx.to_networkx_graph` inline here or in a see also?

Reply:
Discussed in person and decided not to add any link to keep it consistent with rest of docs.